account_fiscal_position_partner_type/views/account_fiscal_position.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright 2021 Valentin Vinagre <valentin.vinagre@sygel.es>
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+    <record id="view_account_position_form_fiscal_position_type" model="ir.ui.view">
+        <field name="name">view.account.position.form.fiscal.position.type</field>
+        <field name="model">account.fiscal.position</field>
+        <field name="inherit_id" ref="account.view_account_position_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='auto_apply']" position="after">
+                <field
+                    name="fiscal_position_type"
+                    attrs="{'invisible': [('auto_apply', '!=', True)]}"
```

Comment:
Why limiting to `auto_apply`? This can help in other use cases apart from OSS. That's why we put it as a separate module.

Reply:
Because the search logic that takes into account the 'fiscal_position_type' field does not apply if the 'auto_apply' field is not activated.

Reply:
I think you shouldn't limit that. This is applying to the method `get_fiscal_position`, being called by auto-applied or whatever.

Reply:
I've done it this way because odoo in the '_get_fpos_by_region' function always searches with the 'auto_apply' field set to true.

https://github.com/odoo/odoo/blob/17e19c6bf787fd2bf77476ffcfa1d934058aa7ec/addons/account/models/partner.py#L126-L130

If the field is displayed without being used in the search, it could lead to think that the module does not work.

Reply:
OK, seems correct to me. On the way to merging!